src/core/server/config/schema/internals/index.ts

```diff
@@ -0,0 +1,304 @@
+/*
```

Comment:
__note:__ if this file continue to grow, I'll probably split it into separate files within `internals/rules` folder.